models/reviews.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require("../db/connection");
+
+exports.selectReviewById = (reviewId) => {
+  if (isNaN(reviewId) === true) {
```

Comment:
[isNaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#:~:text=This%20behavior%20of%20isNaN()%20for%20non%2Dnumeric%20arguments%20can%20be%20confusing!) not least because article_id will be a string from the parameterised endpoint url.
Remember that the database is expecting the column value of article_id to be a number so will give a specific error in the case of article_id being something else.

Look at the error that comes back from the database if you pass anything other than a string-integer to query. You could handle what is returned with a custom error handling middleware in app.js